pkgs/stdenv/adapters.nix

```diff
@@ -251,7 +251,7 @@ rec {
          (self: super: { stdenv = super.withCFlags { stdenv = super.stdenv; compilerFlags = [ "-funroll-loops" "-O3" ]; }; })
        ];
   */
-  withCFlags = { stdenv, compilerFlags }:
+  withCFlags = stdenv: compilerFlags:
```

Comment:
Argument order should be flipped, see e. g. validateLicenses

Reply:
Fixed and rebased.